### ISSUE TITLE
runtime error on server - v0.2.4

### ISSUE THREAD

**Issue by Shleeble** [yukimochi/Activity-Relay#29]
Running on https://neighbours.aus.social/ - Latest release v0.2.4
```
server_1      | 2020/01/29 03:33:01 http: panic serving 172.26.0.1:44626: runtime error: invalid memory address or nil pointer dereference
server_1      | goroutine 285 [running]:
server_1      | net/http.(*conn).serve.func1(0xc000136e60)
server_1      | 	/usr/local/go/src/net/http/server.go:1767 +0x139
server_1      | panic(0xed5640, 0x19a7910)
server_1      | 	/usr/local/go/src/runtime/panic.go:679 +0x1b2
server_1      | github.com/yukimochi/Activity-Relay/KeyLoader.ReadPublicKeyRSAfromString(0x0, 0x0, 0x27, 0xc000084180, 0x5d)
server_1      | 	/Activity-Relay/KeyLoader/keyloader.go:28 +0x69
server_1      | main.decodeActivity(0xc0000eb500, 0x203000, 0x203000, 0x203000, 0x203000, 0x200, 0xe, 0x203000)
server_1      | 	/Activity-Relay/decode.go:35 +0x3f4
server_1      | main.handleInbox(0x12af0e0, 0xc00037e0e0, 0xc0000eb500, 0x105b840)
server_1      | 	/Activity-Relay/handle.go:174 +0xbb
server_1      | main.main.func1(0x12af0e0, 0xc00037e0e0, 0xc0000eb500)
server_1      | 	/Activity-Relay/main.go:102 +0x4b
server_1      | net/http.HandlerFunc.ServeHTTP(0x105b858, 0x12af0e0, 0xc00037e0e0, 0xc0000eb500)
server_1      | 	/usr/local/go/src/net/http/server.go:2007 +0x44
server_1      | net/http.(*ServeMux).ServeHTTP(0x19bfda0, 0x12af0e0, 0xc00037e0e0, 0xc0000eb500)
server_1      | 	/usr/local/go/src/net/http/server.go:2387 +0x1bd
server_1      | net/http.serverHandler.ServeHTTP(0xc00037e000, 0x12af0e0, 0xc00037e0e0, 0xc0000eb500)
server_1      | 	/usr/local/go/src/net/http/server.go:2802 +0xa4
server_1      | net/http.(*conn).serve(0xc000136e60, 0x12b3320, 0xc0000c79c0)
server_1      | 	/usr/local/go/src/net/http/server.go:1890 +0x875
server_1      | created by net/http.(*Server).Serve
server_1      | 	/usr/local/go/src/net/http/server.go:2928 +0x384
```

**Comment by yukimochi** [comment]
Thanks for reporting errors, and sorry for inconvenient error messages.

This error implies remote actor's certificate is invalid. Server raising panic is not desired, it should be suppressed and run continually.

I'll fix this problem and release new version soon.

**Comment by yukimochi** [comment]
Released [v0.2.5](https://github.com/yukimochi/Activity-Relay/releases/tag/v0.2.5).
Please try v0.2.5. If recur yet, please reopen issue.
